ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -85,15 +90,46 @@ gcp_connection::get_object(const std::string &object_key, const std::string &pat
 // Checks whether an object with the given key exists in the google cloud bucket and also retrieves
 // size of the object.
 int
-gcp_connection::object_exists(
-  const std::string &object_key, bool &exists, size_t &object_size) const
+gcp_connection::object_exists(const std::string &object_key, bool &exists, size_t &object_size)
 {
+    object_size = 0;
+
+    google::cloud::StatusOr<gcs::ObjectMetadata> mt =
```

Comment:
The variable name `mt` could be improved. Maybe use `metadata` instead?

Reply:
they have been updated here: [e806560](https://github.com/wiredtiger/wiredtiger/pull/8737/commits/e806560cfcd70355152938a098de63b3c5071635) :)